### PR TITLE
CU-86ctxw2uw - env example and add condition to firebase

### PR DESCRIPTION
**Pull request by vaionexbusra**
**Task**

[Link](https://app.clickup.com/t/86ctxw2uw)

**Implementation** 

Env.local filled with the name of the keys. 
Initialize firebase has added condition. (for firebase, storage and firestore.)

### CODE REVIEW

File: src/utils/stripe/index.js [hunk 1]
```diff
@@ -1,162 +1,92 @@
...
+      }
+
+      plans.push(plan)
+    }
+
+    return plans
   } catch (error) {
-    console.error('Error retrieving pricing plans:', error)
+    console.error('Error retrieving products and plans:', error)
     throw error
```
Comment (mrf1freak) [inline src/utils/stripe/index.js:31]: If we fail to fetch any plans, we return placeholder plans

**Review by mrf1freak — CHANGES_REQUESTED** [review]

File: src/app/api/stripe/checkout/route.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const session = await stripe.checkout.sessions.create({
+      payment_method_types: ['card'],
+      line_items: [
+        {
+          price: priceId,
+          quantity: 1,
+        },
+      ],
+      mode: 'subscription',
+      success_url: 'http://localhost:3000/pricing',
```
Comment (paulsp94) [inline src/app/api/stripe/checkout/route.js:19]: this is definitly wrong and will break in every deployment

File: src/components/Pricing/index.jsx [hunk 3]
```diff
@@ -31,15 +30,14 @@ const comingSoonFeatures = [
...
 
 export default function Products({ plans }) {
-  const { authUser } = useFirebaseAuthContext()
-  const firebaseAuth = useFirebaseAuth()
+  const { authUser } = useAuth()
   const { push } = useRouter()
-
   const [activeTab, setActiveTab] = useState('month')
   const [currentPlan, setCurrentPlan] = useState(plans[0]?.id)
+  // const [currentPlan, setCurrentPlan] = useState('')
```
Comment (paulsp94) [inline src/components/Pricing/index.jsx:37]: remove this we dont want dead code

File: src/utils/stripe/index.js [hunk 4]
```diff
@@ -1,162 +1,168 @@
...
-    //free plan
-    const pricePlanRef = doc(
-      firestore,
-      'products',
-      process.env.NEXT_PUBLIC_STRIPE_FREE_PRODUCT_ID,
-      'prices',
-      process.env.NEXT_PUBLIC_STRIPE_FREE_PRODUCT_PRICE_ID,
-    )
-    const pricePlanSnapshot = await getDoc(pricePlanRef)
+// export const initiateSubscription = async (planProductId, errorHandle) => {
```
Comment (paulsp94) [inline src/utils/stripe/index.js:139]: can we clean this up

**Review by paulsp94 — COMMENTED** [review]
This should have been 2 PRs perhaps, one for stripe and on for firebase

**Pull request merged**